site-src/blog/2023/0823-welcome-gamma.md

```diff
@@ -0,0 +1,160 @@
+- [GEP-1426](https://gateway-api.sigs.k8s.io/geps/gep-1426/) defines how to
+  use Gateway API route resources, such as HTTPRoute, to manage traffic within
+  a service mesh.
+- [GEP-1686](https://gateway-api.sigs.k8s.io/geps/gep-1686/) builds on the
+  work of [GEP-1709](https://gateway-api.sigs.k8s.io/geps/gep-1709/) to define
+  a _conformance profile_ for service meshes to be declared conformant with
+  the Gateway API.
+
+Although these are [Experimental][status] APIs, note that they are available
+in the [`standard` release channel][ch], since the GAMMA initiative has not
+needed to introduce new resources or fields to date.
+
+### How will mesh routing work when using the Gateway API?
+
+You can find all the details in the [Gateway API Mesh routing
+documentation][mesh-routing] and [GEP-1426], but the short version for Gateway
+API v0.8.0 is that an HTTPRoute can now have a `parentRef` that is a Service,
+rather than just a Gateway. We anticipate future GEPs in this area as we gain
+more experience with service mesh use cases -- binding to a Service makes it
+possible to use the Gateway API with a service mesh, but there are several
+interesting use cases that remain difficult to cover.
+
+As an example, you might use an HTTPRoute to do an A-B test in the mesh as
+follows:
+
+```yaml
+  apiVersion: gateway.networking.k8s.io/v1beta1
+  kind: HTTPRoute
+  metadata:
+    name: bar-route
+  spec:
+    parentRefs:
+      - name: demo-app
+          port: 5000
+          kind: Service
+    rules:
+    - matches:
+      - headers:
+        - type: Exact
+          name: env
+          value: v1
+      backendRefs:
+      - name: demo-app-v1
+        port: 5000
+    - backendRefs:
+      - name: demo-app-v2
+        port: 5000
+```
+
+Any request to port 5000 of the `demo-app` Service that has the header `env:
+v1` will be routed to `demo-app-v1`, while any request without that header
+will be routed to `demo-app-v2` -- and since this is being handled by the
+service mesh, not the ingress controller, the A/B test can happen anywhere in
+the application's call graph.
+
+### How does Gateway API conformance work for a service mesh?
+
+One of the challenges that the GAMMA initiative ran into is that Gateway API
+conformance was strongly tied to the idea that a given implementation provides
+an ingress controller. Many service meshes don't, and requiring a
+GAMMA-conformant mesh to also implement an ingress controller seemed
+impractical at best. This resulted in work restarting on Gateway API
+_conformance profiles_, as discussed in
+[GEP-1709](https://gateway-api.sigs.k8s.io/geps/gep-1709/).
+
+The basic idea of conformance profiles is that we can define subsets of the
+Gateway API, and allow implementations to choose - and document! - which
+subsets they conform to. (SIG-Network-Policy is going to be using this concept
+as well.) GAMMA is adding a new profile, named `Mesh`, which checks only the
```

Comment:
Nit: Technically I think network policy is a subproject of SIG-Network, but since I don't think we have a great place to link out to yet, I'd just remove the reference to the network policy.

Reply:
I missed the context on how/why this was mentioned, but I think the recently-launched https://network-policy-api.sigs.k8s.io/ may be appropriate to link in the future if desired?